wcs-functions.php

```diff
@@ -30,16 +30,18 @@
 
 /**
  * Check if a given object is a WC_Subscription (or child class of WC_Subscription), or if a given ID
- * belongs to a post with the subscription post type ('shop_subscription')
+ * belongs to a post with the subscription post type ('shop_subscription').
```

Comment:
When HPOS is enabled, this is not true right? I suggest a change. Feel free to modify.

```suggestion
 * belongs to a post or order with type ('shop_subscription').
```